_posts/randy/2022-06-24-reflecting-on-the-future.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Reflecting on the Future - A Mirror Sample of Thinks to Come"
+permalink: /@randy/reflecting-on-the-future
+ext_image: "assets/img/sample-in-progress.png"
+description: I am Old School human; equipped with analog gears but upgraded with digital belts. Having a fair amount of past, I spend a lot of time reflecting on the past. It seems like I always have a need to measure and gauge my present with the paths and events it took to get me here.
+tags: tails analog-sampling
+---
+
+![](/assets/img/sample-in-progress.png)
+
+<!-- <video width="720" height="480" controls>
+   <source src="/assets/img/BlackHoleSon720.mp4" type="video/mp4">
+</video> -->
+
+I am Old School human; equipped with analog gears but upgraded with digital belts. Having a fair amount of past, I spend a lot of time reflecting on the past. It seems like I always have a need to measure and gauge my present with the paths and events it took to get me here. 
+
+I have a million memories of positive greatness, but the reflections that come back are often on the negative side. Perhaps that is just physics. A mirror, says the Blue Oyster Cult, is a negative space in a frame. We see everything backwards (and upside down, you know). The positive in life was expected; the negative was not. 
```

Comment:
I'm guessing the Blue Oyster Cult reference is to https://www.youtube.com/watch?v=F1HrO2W6w-4 ?  If so, I think we should link.  Or honestly maybe just embed the youtube video.

Reply:
good idea - option for lyrics as well https://www.azlyrics.com/lyrics/blueoystercult/mirrors.html